diff --git a/Makefile.in b/Makefile.in
index 83fc0cb..30b21da 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -24,7 +24,7 @@ MAKENSIS	= @MAKENSIS@
 
 SUBDIRS		= virt-top
 
-all opt depend install:
+all opt depend install-opt install-byte:
 	for d in $(SUBDIRS); do \
 	  $(MAKE) -C $$d $@; \
 	  if [ $$? -ne 0 ]; then exit 1; fi; \
@@ -76,4 +76,4 @@ check-manifest:
 	exit $$rv
 
 .PHONY: all opt depend install clean distclean configure dist check-manifest \
-	release release_stage_2 release_stage_3 force
\ No newline at end of file
+	release release_stage_2 release_stage_3 force
diff --git a/virt-top/Makefile.in b/virt-top/Makefile.in
index e471f93..ad1799c 100755
--- a/virt-top/Makefile.in
+++ b/virt-top/Makefile.in
@@ -95,7 +95,7 @@ virt-top.txt: virt-top.pod
 	pod2text $< > $@
 endif
 
-install:
+install-opt:
 	if [ -x virt-top.opt ]; then \
 	  mkdir -p $(DESTDIR)$(bindir); \
 	  $(INSTALL) -m 0755 virt-top.opt $(DESTDIR)$(bindir)/virt-top; \
